### DISCUSSION TITLE
On the way of fedora

### DISCUSSION THREAD

**Discussion by pieroproietti** [pieroproietti/penguins-eggs-legacy#376] [Announcements]
Introducing fedora, has always been a goal of mine, this time - maybe - the attempts will succeed.

![fedora](https://github.com/user-attachments/assets/8e0fa939-2d66-4525-a179-923d3d6b63cc)

I'm writing the progress on [WAY-TO-FEDORA.md](https://github.com/pieroproietti/penguins-eggs/blob/master/DOCS/WAY-TO-FEDORA.md).

**Comment by mak448a** [comment]
Any updates on this?

**Reply by pieroproietti** [reply]
Un fortunately no. Actually In working on Alpine Linux, on Alpine Im a good point not yet finished.

Fedora will be possible, but Need time and help.

**Reply by pieroproietti** [reply]
![work-in-progress](https://github.com/user-attachments/assets/7d4c48df-853b-495f-a146-d57f38e21941)

Work in progress... today was tired with Alpine and just tryed something different.

Are just two hours of work, nothing special, I just installed the last source version on a minimal fedora - dressed as colibri  following the  way-of-fedora.md on DOCUMENTS....

Can create the ISO, but the live don't start... must to see the reasons and there are a lot of course!

**Comment by focuspointlinux** [comment]
I'm really hoping for this as well.

**Reply by pieroproietti** [reply]
More than an update...

https://sourceforge.net/projects/penguins-eggs/files/ISOS/fedora/

Read the [last post on the blog](https://penguins-eggs.net/blog/way-to-fedora)

A big effort, something to do again, but go!

**Reply by focuspointlinux** [reply]
Thanks a bunch for your work, it's pretty awesome. I hope you will get some help pulling the load

**Comment by pieroproietti** [comment]
I forgot to update here: actually we have `penguins-eggs` on fedora and opensuse.

There are also ISOs on [sourceforge](https://sourceforge.net/projects/penguins-eggs/files/ISOS/fedora/) and is possible to install `penguins-eggs` and remaster fedora, following this [README](https://github.com/pieroproietti/penguins-eggs/tree/master/PREREQUISITES#prerequisites).

Enjoy, and if someone will create an rpm package, will be great!

**Comment by pieroproietti** [comment]
Yesterday, I tested penguins'eggs against the new version [fedora 41](https://fedoramagazine.org/announcing-fedora-linux-41/) with good results. You can find naked (CLI) and colibri (XFCE) versions on [sourceforge](https://sourceforge.net/projects/penguins-eggs/files/ISOS/fedora/). 

To install penguins-eggs on your installation and get a live ISO of your installed system, follow this [README](https://github.com/pieroproietti/penguins-eggs/tree/master/PREREQUISITES#prerequisites).
